examples/basics/src/lib.rs

```diff
@@ -0,0 +1,104 @@
+#[macro_use]
+extern crate neon;
+
+// thanks to @rm-rf-etc for these: https://github.com/neon-bindings/neon/issues/260
+use neon::vm::{Call, JsResult};
+use neon::js::{
+  Object, // <- https://github.com/neon-bindings/neon/issues/57
+  JsFunction,
+  JsObject,
+  JsArray,
+  JsBoolean,
+  JsInteger,
+  JsNull,
+  JsNumber,
+  JsString,
+  JsUndefined};
+
+/*
+	// If you need to inspect the value of an expression:
+	let () = JsBoolean::new();
```

Comment:
This comment is a little confusing: did you mean `()` to be a stand-in for "whatever pattern you want to match"? As is, `()` is real Rust syntax, and this statement as you've written it would parse properly but cause a type error. That seems like it could trip someone up who's trying to understand the API. Maybe something like `let <<put your pattern here>> = JsBoolean::new();`? I guess I'm not completely sure what you're trying to get across in this comment?

Reply:
I'm not sure what was meant by that. I copied most(all?) of that from #260. I can delete it